engine/src/main/java/org/terasology/logic/ai/HierarchicalAISystem.java

```diff
@@ -125,17 +125,17 @@ private void loop(EntityRef entity, LocationComponent location,
 
             // if our AI is aggressive or hunter go and hunt player else run away
             // if wild
-            if (ai.aggressive) {
+            /*if (ai.aggressive) {
```

Comment:
Hmmmm... ideally here we'd track down who commented out the content of the if block. I wonder if there's an easy way to do that via git.